src/test/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/FindOneAndReplaceCommandResolverTest.java

```diff
@@ -234,7 +234,7 @@ public void idFilterConditionWithOptions() throws Exception {
           "findOneAndReplace": {
             "filter" : {"_id" : "id"},
             "replacement" : {"col1" : "val1", "col2" : "val2"},
-            "options" : {"returnDocument" : "after", "upsert": true }
+            "options" : {"returnDocument" : "after" }
```

Comment:
why this removed?

Reply:
As per my comment (not sure if it got removed?), `findOneAndReplace` does not support this option.
With "ignore all unknown" it was let through, but removing that tests flagged it as unknown.

So it's a fix to bug hidden by ignoring unknown properties.

Reply:
Ahaaa good!